addon/mixin.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable ember/no-new-mixins */
```

Comment:
Further reading:
- [`no-new-mixins`](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-new-mixins.md)
- https://github.com/emberjs/rfcs/issues/534

Reply:
Yeah, I thought about deprecating it, but figured it's best to have a version that has both the modifier and the mixin. I'll get rid of the mixin from `2.0`

Reply:
Yeah, that makes sense to me, it's handy to have both for now!